site/gatsby-site/src/components/incidents/IncidentForm.js

```diff
@@ -16,6 +32,40 @@ export const schema = Yup.object().shape({
 function IncidentForm() {
   const { values, errors, handleChange, handleSubmit } = useFormikContext();
 
+  const [editorSimilarIncidentReports, setEditorSimilarIncidentReports] = useState([]);
+
+  const [similarReportsById, setSimilarReportsById] = useState([]);
+
+  const client = useApolloClient();
+
+  const updateEditorSimilarIncidents = async () => {
+    const dbResponse = await client.query({
+      query: relatedIncidentIdsQuery,
+      variables: {
+        query: {
+          incident_id_in: (values?.editor_similar_incidents || []).concat(
+            values.editor_dissimilar_incidents
+          ),
+        },
+      },
+    });
+
+    const reports = dbResponse.data.incidents.reduce(
+      (reports, incident) =>
+        reports.concat(
+          incident.reports.map((report) => ({ ...report, incident_id: incident.incident_id }))
+        ),
+      []
+    );
+
+    setEditorSimilarIncidentReports(reports);
+  };
+
+  useEffect(() => {
+    updateEditorSimilarIncidents();
+    window.location.hash && document.querySelector(window.location.hash).scrollIntoView();
```

Comment:
Gatsby already has a scroll restoration feature, and I'm not sure you want this to happen everywhere the incident form is used (present and maybe future uses).

Reply:
Prior to adding this, I was finding that linking to the `#similar-incidents` anchor on the IncidentForm was not working. I'm not sure of a better way to do this.

Reply:
Yeah, that should work without this. Created https://github.com/responsible-ai-collaborative/aiid/issues/811, so we don't forget.